core-services/prow/02_config/_plugins.yaml

```diff
@@ -809,19 +809,19 @@ config_updater:
         app.ci:
         - ci
       gzip: true
-      name: job-config-master
+      name: job-config-master-periodics
```

Comment:
That should be ` job-config-main-periodics`, right? Otherwise, it will conflict with https://github.com/openshift/release/pull/22494/files#diff-a9035bbaee5af66130c81cfd0b97df589a7d2e22e87ad9d3800851407967c07bR854

Reply:
No unless we want to shard master and main. These files previously went to `job-config-master`. There's no `job-config-main-periodics`.

Reply:
It will never conflict, because the keys coming from this stanza will always be `*-main-periodics.yaml` and the keys from the L849 stanza will always be `-master-periodics.yaml`.